torchvision/csrc/vision.cpp

```diff
@@ -1,55 +1,22 @@
-#include <Python.h>
-#include <torch/script.h>
+// Autoregister all the torchvision ops (for C++).
+#include "ops_autoregister.h"
 
-#ifdef WITH_CUDA
-#include <cuda.h>
-#endif
-#ifdef WITH_HIP
-#include <hip/hip_runtime.h>
-#endif
-
-#include "DeformConv.h"
-#include "PSROIAlign.h"
-#include "PSROIPool.h"
-#include "ROIAlign.h"
-#include "ROIPool.h"
-#include "empty_tensor_op.h"
-#include "nms.h"
+#include <Python.h>
 
 // If we are in a Windows environment, we need to define
 // initialization functions for the _C extension
 #ifdef _WIN32
 #if PY_MAJOR_VERSION < 3
 PyMODINIT_FUNC init_C(void) {
-  // No need to do anything.
-  // extension.py will run on load
+  // Register torchvision ops for python.
+  vision::RegisterOps();
   return NULL;
 }
 #else
 PyMODINIT_FUNC PyInit__C(void) {
```

Comment:
Those functions will be executed only when you call `import torchvision._C`, not that you load them explicitly through `ctypes.CDLL`. I guess you'll need to move the function call to the global space.